src/System.Threading.AccessControl/tests/SemaphoreAclTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using Xunit;
+
+namespace System.Threading.Tests
+{
+    public class SemaphoreAclTests : AclTests
+    {
+        private int _defaultInitialCount = 0;
+        private int _defaultMaximumCount = 1;
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
That's an interesting question. Let me run the tests without the headers.
The reason why I added the attribute is because the existing tests in the ThreadingAclExtensionsTests.cs file had that header.

Reply:
They passed.